### ISSUE TITLE
RG353M disconnects wifi on sleep

### ISSUE THREAD

**Issue by Razzly** [TheGammaSqueeze/GammaOS-RK3566#4]
When device goes to sleep wifi dissconnects regardless of options selected.

Installed full GammaOS.

**Comment by theredkoolaid** [comment]
I would have the same thing happen with factory os. Could be how the device goes into hibernation mode?

**Comment by Razzly** [comment]
Yeah, seems that way. I tried to change battery settings for WiFi from optimized to full stream, but nothing improved. This is a Little problematic specially if i want to download something, the i need to change screen on settings to ex. 30min to help WiFi stay on.

There used to be special settings to keep WiFi turned on while sleep but they removed it some Time ago from android...

**Comment by farmerbb** [comment]
I haven't tried changing these yet on my RG ARC-D, but I found two settings at the bottom of the build.prop that might control the Wi-Fi disconnection behavior on sleep:

```
ro.wifi.sleep.power.down=true
persist.wifi.sleep.delay.ms=0
```
